### ISSUE TITLE
IIIF access should only happen via Curate

### ISSUE THREAD

**Issue by maxkadel** [emory-libraries/dlp-lux#391]
We need to ensure that no one can go directly to our Cantaloupe servers. Traffic directly to Cantaloupe will not be protected by the image restrictions that Curate has in place. 

**Acceptance Criteria**
- [ ] Only Curate servers (curate.library.emory.edu ; curate-arch.library.emory.edu ; curate-test.library.edu) can request image from Cantaloupe servers (iiif-cor-arch, etc.). 
![image.png](https://images.zenhubusercontent.com/5d51624a5b89150001ae0b03/07e10c5f-fc59-4c68-b6ad-c325a55d698c)

**Comment by SolomonHD** [comment]
Today I updated the security groups of all instances and restricted iiif to only be able to communicate with curate. 

The curate app seems to be serving the images but it seems that curate and lux are not properly coded to handle proxied iiif requests because when the restrictions are in place neither curate or lux display images.

Currently I've returned iiif-cor-arch and iiif-cor to having relaxed permissions and being able to talk to entire internet. iiif-cor-test is still under full restrictions.

Evidence that curate-test is proxing images:
https://curate-test.library.emory.edu/iiif/2/942a591396db80fe440079d47fd8994d5d4debcb/full/600,/0/default.jpg
